app/src/testFull/kotlin/io/homeassistant/companion/android/sensors/HealthConnectSensorManagerTest.kt

```diff
@@ -0,0 +1,58 @@
+package io.homeassistant.companion.android.sensors
+
+import android.content.Context
+import androidx.health.connect.client.HealthConnectClient
+import androidx.health.connect.client.HealthConnectFeatures
+import androidx.health.connect.client.permission.HealthPermission
+import androidx.health.connect.client.testing.ExperimentalTestingApi
+import androidx.health.connect.client.testing.FakeHealthConnectClient
+import io.homeassistant.companion.android.testing.unit.ConsoleLogTree
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkObject
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+import timber.log.Timber
+
+@OptIn(ExperimentalTestingApi::class)
+class HealthConnectSensorManagerTest {
+
+    private val sensorManager = HealthConnectSensorManager()
+
+    private val context = mockk<Context> {
+        every { applicationContext } returns this
+    }
+    private val healthConnectClient = FakeHealthConnectClient()
```

Comment:
To be fair seeing the current state of the API, I would remove this alpha deps and simply use a mock of `HealtConnectClient` which is going to result in almost the same code since you already have to mockkObject (even in alpha3)